<ui version="4.0" >
 <class>SetupWidget</class>
 <widget class="QWidget" name="SetupWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>313</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="playerBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Players</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" >
      <item row="2" column="1" >
       <widget class="KComboBox" name="blackPlayerCombo" >
        <item>
         <property name="text" >
          <string>Human</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Computer</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="2" colspan="2" >
       <widget class="QStackedWidget" name="blackPlayerStack" >
        <property name="currentIndex" >
         <number>0</number>
        </property>
        <widget class="QWidget" name="blackHumanPage" >
         <layout class="QGridLayout" name="gridLayout_2" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item row="0" column="0" >
           <widget class="KLineEdit" name="blackPlayerName" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="text" >
             <string>Name</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="blackComputerPage" >
         <layout class="QGridLayout" name="gridLayout_3" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item row="0" column="4" >
           <widget class="QLabel" name="label2_3" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Strong</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QLabel" name="label1_3" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Weak</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="3" >
           <widget class="QSlider" name="blackStrengthSlider" >
            <property name="minimum" >
             <number>1</number>
            </property>
            <property name="maximum" >
             <number>10</number>
            </property>
            <property name="pageStep" >
             <number>1</number>
            </property>
            <property name="value" >
             <number>5</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition" >
             <enum>QSlider::TicksBelow</enum>
            </property>
            <property name="tickInterval" >
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="2" colspan="2" >
       <widget class="QStackedWidget" name="whitePlayerStack" >
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="currentIndex" >
         <number>0</number>
        </property>
        <widget class="QWidget" name="whiteHumanPage" >
         <layout class="QHBoxLayout" name="horizontalLayout_2" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="KLineEdit" name="whitePlayerName" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="text" >
             <string>Name</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="whiteComputerPage" >
         <layout class="QGridLayout" name="gridLayout_8" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="label1" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Weak</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="3" >
           <widget class="QLabel" name="label2" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Strong</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QSlider" name="whiteStrengthSlider" >
            <property name="minimum" >
             <number>1</number>
            </property>
            <property name="maximum" >
             <number>10</number>
            </property>
            <property name="pageStep" >
             <number>1</number>
            </property>
            <property name="value" >
             <number>5</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition" >
             <enum>QSlider::TicksBelow</enum>
            </property>
            <property name="tickInterval" >
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>White:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>whitePlayerCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Black:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>blackPlayerCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="KComboBox" name="whitePlayerCombo" >
        <item>
         <property name="text" >
          <string>Human</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Computer</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
     <zorder>blackPlayerCombo</zorder>
     <zorder>blackPlayerStack</zorder>
     <zorder>whitePlayerStack</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>whitePlayerCombo</zorder>
     <zorder>gameSetupStack</zorder>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QStackedWidget" name="gameSetupStack" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="newGamePage" >
      <layout class="QVBoxLayout" name="verticalLayout_7" >
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame" >
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="newGameBox" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="title" >
             <string>New game</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_7" >
             <item rowspan="2" row="0" column="0" >
              <widget class="KButtonGroup" name="sizeGroupBox" >
               <property name="title" >
                <string>Board size</string>
               </property>
               <property name="flat" >
                <bool>false</bool>
               </property>
               <property name="current" stdset="0" >
                <number>2</number>
               </property>
               <layout class="QGridLayout" name="gridLayout_10" >
                <item row="0" column="0" colspan="2" >
                 <widget class="QRadioButton" name="sizeSmall" >
                  <property name="text" >
                   <string>Tiny (9x9)</string>
                  </property>
                  <property name="checked" >
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" colspan="2" >
                 <widget class="QRadioButton" name="sizeMedium" >
                  <property name="text" >
                   <string>Small (13x13)</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0" colspan="2" >
                 <widget class="QRadioButton" name="sizeBig" >
                  <property name="text" >
                   <string>Normal (19x19)</string>
                  </property>
                  <property name="checked" >
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0" >
                 <widget class="QRadioButton" name="sizeOther" >
                  <property name="text" >
                   <string>Other:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1" >
                 <widget class="QSpinBox" name="sizeOtherSpinBox" >
                  <property name="enabled" >
                   <bool>false</bool>
                  </property>
                  <property name="minimum" >
                   <number>5</number>
                  </property>
                  <property name="maximum" >
                   <number>19</number>
                  </property>
                  <property name="value" >
                   <number>19</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="1" >
              <widget class="QGroupBox" name="groupBox_3" >
               <property name="title" >
                <string>Komi</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_8" >
                <item>
                 <widget class="QDoubleSpinBox" name="komiSpinBox" >
                  <property name="suffix" >
                   <string> Points</string>
                  </property>
                  <property name="decimals" >
                   <number>1</number>
                  </property>
                  <property name="maximum" >
                   <double>50.000000000000000</double>
                  </property>
                  <property name="value" >
                   <double>4.500000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="1" >
              <widget class="QGroupBox" name="handicapGroupBox" >
               <property name="title" >
                <string>Fixed handicap</string>
               </property>
               <property name="flat" >
                <bool>false</bool>
               </property>
               <property name="checkable" >
                <bool>true</bool>
               </property>
               <property name="checked" >
                <bool>false</bool>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_9" >
                <item>
                 <widget class="QSpinBox" name="handicapSpinBox" >
                  <property name="suffix" >
                   <string> Stones</string>
                  </property>
                  <property name="minimum" >
                   <number>2</number>
                  </property>
                  <property name="maximum" >
                   <number>9</number>
                  </property>
                  <property name="value" >
                   <number>4</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="loadGamePage" >
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2" >
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item row="0" column="0" >
           <widget class="QGroupBox" name="loadedGameBox" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="title" >
             <string>Loaded game</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5" >
             <item row="0" column="1" colspan="3" >
              <widget class="KIntSpinBox" name="startMoveSpinBox" >
               <property name="suffix" >
                <string> of 999</string>
               </property>
               <property name="prefix" >
                <string>Move </string>
               </property>
               <property name="minimum" >
                <number>1</number>
               </property>
               <property name="maximum" >
                <number>9999</number>
               </property>
              </widget>
             </item>
             <item row="4" column="3" colspan="2" >
              <widget class="KLineEdit" name="infoLocation" >
               <property name="enabled" >
                <bool>true</bool>
               </property>
               <property name="focusPolicy" >
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="text" >
                <string/>
               </property>
               <property name="readOnly" >
                <bool>true</bool>
               </property>
               <property name="contextMenuEnabled" >
                <bool>false</bool>
               </property>
               <property name="urlDropsEnabled" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="1" colspan="2" >
              <widget class="KLineEdit" name="infoDate" >
               <property name="enabled" >
                <bool>true</bool>
               </property>
               <property name="focusPolicy" >
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="text" >
                <string/>
               </property>
               <property name="maxLength" >
                <number>10</number>
               </property>
               <property name="readOnly" >
                <bool>true</bool>
               </property>
               <property name="contextMenuEnabled" >
                <bool>false</bool>
               </property>
               <property name="urlDropsEnabled" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="1" colspan="4" >
              <widget class="KLineEdit" name="infoEvent" >
               <property name="enabled" >
                <bool>true</bool>
               </property>
               <property name="focusPolicy" >
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="text" >
                <string/>
               </property>
               <property name="readOnly" >
                <bool>true</bool>
               </property>
               <property name="contextMenuEnabled" >
                <bool>false</bool>
               </property>
               <property name="urlDropsEnabled" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0" >
              <widget class="QLabel" name="label_8" >
               <property name="text" >
                <string>Event:</string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="4" column="0" >
              <widget class="QLabel" name="label_9" >
               <property name="text" >
                <string>Location:</string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="5" >
              <widget class="Line" name="line_2" >
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="5" column="3" >
              <widget class="QLabel" name="label_10" >
               <property name="text" >
                <string>Score:</string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="5" column="0" >
              <widget class="QLabel" name="label_11" >
               <property name="text" >
                <string>Round:</string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="5" column="4" >
              <widget class="KLineEdit" name="infoScore" >
               <property name="enabled" >
                <bool>true</bool>
               </property>
               <property name="focusPolicy" >
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="text" >
                <string/>
               </property>
               <property name="readOnly" >
                <bool>true</bool>
               </property>
               <property name="contextMenuEnabled" >
                <bool>false</bool>
               </property>
               <property name="urlDropsEnabled" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="5" column="1" colspan="2" >
              <widget class="KLineEdit" name="infoRound" >
               <property name="enabled" >
                <bool>true</bool>
               </property>
               <property name="focusPolicy" >
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="text" >
                <string/>
               </property>
               <property name="maxLength" >
                <number>3</number>
               </property>
               <property name="readOnly" >
                <bool>true</bool>
               </property>
               <property name="contextMenuEnabled" >
                <bool>false</bool>
               </property>
               <property name="urlDropsEnabled" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="4" >
              <widget class="QLineEdit" name="playerLabel" >
               <property name="focusPolicy" >
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="contextMenuPolicy" >
                <enum>Qt::NoContextMenu</enum>
               </property>
               <property name="acceptDrops" >
                <bool>false</bool>
               </property>
               <property name="text" >
                <string>for Black</string>
               </property>
               <property name="readOnly" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0" >
              <widget class="QLabel" name="label_3" >
               <property name="text" >
                <string>Continue</string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="buddy" >
                <cstring>startMoveSpinBox</cstring>
               </property>
              </widget>
             </item>
            </layout>
            <zorder>startMoveSpinBox</zorder>
            <zorder>infoLocation</zorder>
            <zorder>infoDate</zorder>
            <zorder>infoEvent</zorder>
            <zorder>label_8</zorder>
            <zorder>label_9</zorder>
            <zorder>line_2</zorder>
            <zorder>label_3</zorder>
            <zorder>line</zorder>
            <zorder>label_10</zorder>
            <zorder>label_11</zorder>
            <zorder>infoScore</zorder>
            <zorder>infoRound</zorder>
            <zorder>playerLabel</zorder>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KButtonGroup</class>
   <extends>QGroupBox</extends>
   <header>kbuttongroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>whitePlayerCombo</tabstop>
  <tabstop>blackPlayerCombo</tabstop>
  <tabstop>whitePlayerName</tabstop>
  <tabstop>blackPlayerName</tabstop>
  <tabstop>whiteStrengthSlider</tabstop>
  <tabstop>blackStrengthSlider</tabstop>
  <tabstop>sizeSmall</tabstop>
  <tabstop>sizeMedium</tabstop>
  <tabstop>sizeBig</tabstop>
  <tabstop>sizeOther</tabstop>
  <tabstop>sizeOtherSpinBox</tabstop>
  <tabstop>komiSpinBox</tabstop>
  <tabstop>handicapGroupBox</tabstop>
  <tabstop>handicapSpinBox</tabstop>
  <tabstop>startMoveSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>blackPlayerCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>blackPlayerStack</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>144</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel" >
     <x>278</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>whitePlayerCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>whitePlayerStack</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>144</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel" >
     <x>278</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
